charts/consul/values.yaml

```diff
@@ -557,7 +557,7 @@ global:
   # The name (and tag) of the consul-dataplane Docker image used for the
   # connect-injected sidecar proxies and mesh, terminating, and ingress gateways.
   # @default: hashicorp/consul-dataplane:<latest supported version>
-  imageConsulDataplane: "hashicorppreview/consul-dataplane:1.1-dev"
+  imageConsulDataplane: "docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.2-dev"
```

Comment:
just my own curiosity, does `"docker.mirror.hashicorp.services/hashicorppreview/consul-dataplane:1.2-dev"` == "hashicorppreview/consul-dataplane:1.2-dev"?

Or do you need to use the mirror explicitly to make it the `mirror`